src/sass/Components/_Home.scss

```
@@ -0,0 +1,4 @@
+* {
+	color: white;
```

```suggestion
  color: white;
```

The outputted CSS file seem to follow the 2 space convention, I think we should keep that same styling for the SCSS files.

---

isnt the sass file in src? i named it sass not scss

---

Sorry, my last comment was uncleared. Can we change the name of the directory from `sass` to `scss`? SCSS and SASS uses different syntax and by having a `sass` directory it implies we're following the SASS syntax.